src/core/configs/node.rs

```diff
@@ -349,6 +349,7 @@ pub struct Node {
     cleanup_interval: String,
     open_blobs_soft_limit: Option<usize>,
     open_blobs_hard_limit: Option<usize>,
+    init_par_degree: Option<usize>,
```

Comment:
It's better to use attr `#[serde(default = "default_fn")]` on struct field. And set the default value to 1